appveyor.yml

```diff
@@ -39,9 +39,11 @@ before_build:
 
 build_script:
   - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/s4nhbp52qrknf40m/artifacts/tlf_1.2.43.zip', repos = NULL, type = 'binary')"
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
   - Rscript -e "install.packages('covr', repos = 'http://cran.us.r-project.org')"
+  - Rscript -e "install.packages(c('ggplot2', 'jsonlite', 'reshape2'), repos = 'http://cran.us.r-project.org')"

```

Comment:
any reason why the covr package above is installed separately and not just included into the packages list here?